packages/app-builder-lib/templates/linux/after-install.tpl

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-# Link to the binary
-ln -sf '/opt/${sanitizedProductName}/${executable}' '/usr/bin/${executable}'
+if type update-alternatives 2>/dev/null >&1; then
+    # Remove previous link if it doesn't use update-alternatives
+    if [ -L '/usr/bin/${executable}' -a -e '/usr/bin/${executable}' -a "`readlink '/usr/bin/${executable}'`" != '/etc/alternatives/${executable}' ]; then
```

Comment:
I don't know what this line means, but I trust 😅

Reply:
No worries :)

https://ss64.com/bash/test.html

`-L` - file is symlink.
`EXPR -a EXPR` - AND operation between expressions.
`-e` - check if file exists.

Should result in:
if ( isSymLink() && isExist() && readlink(file) != "/etc/alternatives/$exec" )